docs/src/components/GuidelinesSideBySide.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import { CheckCircle, CloseCircle } from "@kiwicom/orbit-components/icons";
+import { Stack } from "@kiwicom/orbit-components";
+import useMediaQuery from "@kiwicom/orbit-components/lib/hooks/useMediaQuery";
+
+import { GuidelineType } from "./Guideline";
+
+interface GuidelineItemProps extends GuidelineType {
+  children: React.ReactNode;
+  type: "do" | "dont";
+}
+
+const Wrapper = styled.div<GuidelineType>`
+  ${({ theme, type }) => css`
+    border-top: 4px solid
+      ${type === "do" ? theme.orbit.paletteGreenNormal : theme.orbit.paletteRedNormal};
+    padding: ${theme.orbit.spaceLarge} ${theme.orbit.spaceMedium};
+  `}
+`;
+
+interface StyledContainerProps {
+  sizeCheck: boolean | null | undefined;
+}
+
+const StyledContainer = styled.div<StyledContainerProps>`
+  ${({ sizeCheck, theme }) => css`
+    padding: 0 ${theme.orbit.spaceLarge};
+    background-color: ${theme.orbit.paletteCloudLight};
+    border-radius: 16px;
+    width: ${sizeCheck ? "50%" : "100%"};
+  `}
+`;
+
+const GuidelineItem = ({ children, type }: GuidelineItemProps) => (
+  <Stack direction="row" spacing="small" spaceAfter="small">
+    {type === "do" ? (
+      <CheckCircle color="success" ariaLabel="Do" />
+    ) : (
+      <CloseCircle color="critical" ariaLabel="Don't" />
+    )}
+    <div>{children}</div>
+  </Stack>
+);
+
+const GuidelineContainer = ({ children, type }) => {
+  const { isTablet } = useMediaQuery();
+  return (
+    <StyledContainer sizeCheck={isTablet}>
```

Comment:
probably will be better to rename `sizeCheck` to `isTablet`, then it's easier to understand when looking inside SC.

Reply:
Good point. I renamed them in all the files. It came from early testing when I was using different checks for different parts, but in the end, each file has just one check.